>From 0b53a5df8297337c50d2aab067598cfd2d6e2159 Mon Sep 17 00:00:00 2001
From: Chris Lamb <lamby@debian.org>
Date: Sun, 29 Oct 2017 18:54:53 +0000
Subject: [PATCH] debuild: Print the actual arguments we make to lintian.

We are already outputting a line anyway and it's often quite informative
to know whether we "should" see (for example) --pedantic tags, etc.

Signed-off-by: Chris Lamb <lamby@debian.org>
---
 scripts/debuild.pl | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/scripts/debuild.pl b/scripts/debuild.pl
index 2b836dfa..fcaac7eb 100755
--- a/scripts/debuild.pl
+++ b/scripts/debuild.pl
@@ -1055,9 +1055,10 @@ else {
     if ($run_lintian && $lintian_exists) {
 	$<=$>=$uid;  # Give up on root privileges if we can
 	$(=$)=$gid;
-	print "Now running lintian...\n";
-	system('lintian', @lintian_extra_opts, @lintian_opts, $changes);
-	print "Finished running lintian.\n";
+	my @lintian = ('lintian', @lintian_extra_opts, @lintian_opts, $changes);
+	print "Now running @lintian ...\n";
+	system(@lintian);
+	print "Finished running @lintian\n";
     }
 
     # They've insisted.  Who knows why?!
-- 
2.15.0.rc2

